app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1><%= @recipe.name %></h1>
+<h3>Complexity: <%= @recipe.complexity %></h3>
+<h3>For: <%= @recipe.genre %></h3>
+<% @ingredients.each do |ingredient| %>
```

Comment:
Here is where you would want to slim your controller down and call:
```ruby
@recipe.ingredients.each do
```
And we only need the name.